app/views/static_pages/accessibility_statement.html.erb

```diff
@@ -88,47 +89,75 @@
     </p>
 
     <p class="govuk-body">
-      This website is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard, due to
-      the non-compliances listed below.
+      This website is partially compliant with the Web Content Accessibility Guidelines version 2.1 A and AA standards,
```

Comment:
Yeah it does supercede them both, however not all the things in `A` are included in `AA` as its implied that you have met all the `A` criteria first. As this issue explicitly effects us meeting the lower `A` standard I thought it would worthwhile explicitly stating that. 

Not precious though if anyone has any strong feelings?

Reply:
No strong feelings. Just thought it was worth pointing out.